sys/include/net/gnrc/ipv6/nib.h

```diff
@@ -288,6 +288,24 @@ enum {
     GNRC_IPV6_NIB_ROUTE_INFO_TYPE_NSC,
 };
 
+/**
+ * @brief   Types for GNRC_NETIF_BUS_NIB per-interface message bus
+ */
+typedef enum {
+    /**
+     * @brief   Address becomes valid
+     *
+     * The event is generated when an address on the interface becomes valid.
```

Comment:
Maybe also add a note, that this does not fire, if the address is added manually with VALID set.

Reply:
I can also add an event to `gnrc_netif_ipv6_addr_add_internal()`

Reply:
If you think that is not too complicated. I think its not worth it. The user usually knows if they add a valid address or not ;-)

Reply:
Yea but if something wait for a global address to become available, the user configuring a global address should trigger that too.